bokeh/models/plots.py

```diff
@@ -164,7 +164,7 @@ def add_glyph(self, source_or_glyph, glyph=None, **kw):
     y_mapper_type = Either(Auto, String)
 
     extra_x_ranges = Dict(String, Instance(Range1d))
-    extra_y_ranges = Dict(String, Instance(Range1d))
+    extra_y_ranges = Dict(String, Instance(Range))
```

Comment:
`FactorRange` doesn't inherit from `Range1d`. I don't know if it makes sense, but it seems so.  @bryevdv, will know for sure.